### ISSUE TITLE
First style dropped from CSS Document

### ISSUE THREAD

**Issue by JRG** [modxbot/migrate#2865]
JRG created Redmine issue ID 2865
# The first style in in a css document does not appear to make it to the browser (checked with Firebug). Adding a dummy first style (thus turning the original first style into the second one) allows it (the second one) to show up and function. The first part of the CSS document reads as follows (the "div#xxx" is the dummy first style):

/\* Copyright 2004-2010 all rights reserved - Women’s Club of Costa Rica _/
/_ Release 2.2 _/
/_ Layout */
div#xxx {
  background-color: #FFFFCC;
}
div#masthead {
  position: fixed; top: 0; right: 0; height: 9.8em; left: 0;
  overflow: hidden; z-index: 10;
  background-color: #FFFFCC;
  border-bottom: 0.2em solid #E08000;
}
div#copyright   {position: fixed; bottom: 0; height: 1em; right: 0; bottom: 0; left: 0;
                overflow: hidden; z-index: 10;
#                 background-color: #FFFFCC;}

The CSS document is referenced in a chunk (called pageHeader) that is included in a template. The reference is in the HTML <head> section. That complete tag is as follows (all code is copy-and-pasted, not retyped):
# 

<head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
    <title>[[++site_name]] - [[*pagetitle]]</title>
    <base href="[[++site_url]]" />
    <link rel="stylesheet" type="text/css" href="[[~7]]" />
# </head>

**Comment by JRG** [comment]
JRG submitted:

Note that the "==="s are not part of the code. They were meant to delimit it but submission appears to have played with the newlines.

**Comment by splittingred** [comment]
splittingred submitted:

Not sure what this has to do with MODX...

**Comment by JRG** [comment]
JRG submitted:

What it has to do with MODX is that the style sheet works outside of MODX; however, when the style sheet is stored in the MODX database (i.e. as a CSS resource), MODX manages to do something to it so that the first style is not interpreted.

As I said in the post: "... in a css document...".

If you want to sluff-off bug reports... fine.

**Comment by splittingred** [comment]
splittingred submitted:

We're not sluffing-off bug reports; this is not reproducable in any way, shape or form. Maybe you could be more detailed as to how you setup this Resource, and what your server environment is like? 

Also, you should upgrade to the latest version of MODX Revolution as well.

**Comment by JRG** [comment]
JRG submitted:

Shaun McCormick wrote:

> Not sure what this has to do with MODX...

That doesn't sound like "Not reproducible". If you don't want people thinking you are sluffing off bug reports, then write the facts in your responses to bug reports. Closing a bug report as "Invalid" with the comment you made does not exactly imply the bug report was investigated.

I reported the bug against the release that I was running when I encountered it. At the time, it was current. The live site is currently running 2.0.7.
